dbt/adapters/athena/connections.py

```diff
@@ -213,7 +213,12 @@ def format(self, operation: str, parameters: Optional[List[str]] = None) -> str:
             raise ProgrammingError("Query is none or empty.")
         operation = operation.strip()
 
-        if operation.upper().startswith(("SELECT", "WITH", "INSERT")):
+        operation_upper = operation.upper()
+        if (
+            operation_upper.startswith("SELECT")
+            or operation_upper.startswith("WITH")
+            or operation_upper.startswith("INSERT")
+        ):
```

Comment:
how this change is any different than the original implementation??

to me this `operation.upper().startswith(("SELECT", "WITH", "INSERT"))` is evaluated equally to what you propose.

Reply:
Yeah this is the same as the code already in place.

Reply:
It isn't, I just didn't read carefully enough 🤦 
I just saw the issue #60 open and copied my fix from a different fork for that issue, didn't see that the `INSERT` was added.

Sorry for the noise! 

TIL that `startswith` can take a tuple as an argument.

Reply:
No worry highly appreciated anyway.